src/web-animations-next-player.js

```diff
@@ -47,6 +47,63 @@
 
       // FIXME: move existing currentTime/startTime/playState to new player
     },
+    _updateChildren: function() {
+      if (!this.source || this.playState == 'idle')
+        return;
+
+      var offset = this.source._timing.delay;
+      this._childPlayers.forEach(function(childPlayer) {
+        this._arrangeChildren(childPlayer, offset);
+        if (this.source instanceof window.AnimationSequence)
+          offset += scope.groupChildDuration(childPlayer.source);
+      }.bind(this));
+    },
+    _setExternalPlayer: function(player) {
+      if (!this.source || !this._isGroup)
+        return;
+      for (var i = 0; i < this.source.children.length; i++) {
+        this.source.children[i].player = player;
+        this._childPlayers[i]._setExternalPlayer(player);
+      }
+    },
+    _constructChildren: function() {
+      if (!this.source || !this._isGroup)
+        return;
+      var offset = this.source._timing.delay;
+      this.source.children.forEach(function(child) {
+        var childPlayer = window.document.timeline.play(child);
+        this._childPlayers.push(childPlayer);
+        childPlayer.playbackRate = this.playbackRate;
+        if (this.paused)
+          childPlayer.pause();
+        child.player = this.source.player;
+
+        this._arrangeChildren(childPlayer, offset);
+
+        if (this.source instanceof window.AnimationSequence)
+          offset += scope.groupChildDuration(child);
+      }.bind(this));
+    },
+    _arrangeChildren: function(childPlayer, offset) {
+      if (childPlayer.startTime != this.startTime + offset) {
```

Comment:
This still has the same problem. If this.startTime is null, then this.startTime  + offset = offset, which could accidentally be the same as childPlayer.startTime. But if this.startTime is null then we want to set childPlayer.startTime to null.